<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>521</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="login_pushButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>310</y>
        <width>240</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border:2px groove rgb(255,255,255,0,0);border-radius:10px;padding:2px 4px;
	font: 12pt &quot;Adobe 黑体 Std R&quot;;
color:white;
	
	background-color: rgb(85, 85, 255);
}

</string>
      </property>
      <property name="text">
       <string>登陆</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>202</y>
        <width>34</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线 Light</family>
        <pointsize>10</pointsize>
        <weight>6</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 50 10pt &quot;等线 Light&quot;;</string>
      </property>
      <property name="text">
       <string>账户</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="user_password_lineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>237</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;等线&quot;;
border:2px groove gray;border-radius:4px;padding:2px 2px;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>密码</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>240</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 50 10pt &quot;等线 Light&quot;;</string>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_register">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>197</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{background-color: rgb(255, 255, 255,0,0);color:rgb(15,154,232);font-size:15px;}
QPushButton:hover{color:rgb(127,204,248);
background-color:rgb(255,255,255,0,0);}
</string>
      </property>
      <property name="text">
       <string>注册 (&amp;R)</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkbox_remember">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>237</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;等线&quot;;
color:rgb(15,154,232)</string>
      </property>
      <property name="text">
       <string>记住密码</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="uesr_name_lineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>197</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;等线&quot;;
border:2px groove gray;border-radius:4px;padding:2px 2px;</string>
      </property>
      <property name="placeholderText">
       <string>用户名</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>瞎扯问答系统</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_login_image">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/Resource/head_pic.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>521</width>
        <height>371</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="title">
       <string>用户注册</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QLineEdit" name="lineEdit_register_password">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>217</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 11pt &quot;等线&quot;;
border-radius:4px;padding:2px 2px;
background-color: rgb(255, 255, 255,0%);</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>密码</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_register_username">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>175</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 11pt &quot;等线&quot;;
border-radius:4px;padding:2px 2px;
background-color: rgb(255, 255, 255,0%);</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>用户名</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>222</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
       </property>
       <property name="text">
        <string>密码</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>181</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
       </property>
       <property name="text">
        <string>账户</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_register_image">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>60</y>
         <width>81</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/Resource/head_pic.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_register_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>290</y>
         <width>240</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border:2px groove rgb(255,255,255,0,0);border-radius:10px;padding:2px 4px;
	font: 12pt &quot;Adobe 黑体 Std R&quot;;
color:white;
	
	background-color: rgb(85, 85, 255);
}

</string>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_prev_image">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>90</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/Resource/left.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_next_image">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>90</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/Resource/right.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_quit_register">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>10</y>
         <width>30</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/Resource/close.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>195</y>
         <width>211</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>236</y>
         <width>211</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
